istio/supported_ops.go

```diff
@@ -25,6 +25,17 @@ type supportedOperation struct {
 }
 
 const (
+<<<<<<< HEAD
```

Comment:
Can you please remove this line? 
Looks like you committed a conflict. Can you please resolve the conflict?